src/array/cuda/csr_mm.cu

```diff
@@ -6,21 +6,21 @@
 #include <dgl/array.h>
 #include <dgl/runtime/device_api.h>
 
+#include <limits>
+
 #include "../../runtime/cuda/cuda_common.h"
 #include "./cusparse_dispatcher.cuh"
 #include "./functor.cuh"
-
 namespace dgl {
 
 using namespace dgl::runtime;
 
 namespace aten {
 namespace cusparse {
 
-#if 0  // disabling CUDA 11.0+ implementation for now because of problems on
-       // bigger graphs
+#if __CUDACC_VER_MAJOR__ == 12
```

Comment:
Would it be better to use the runtime macro `CUDART_VERSION`? But to be honest, I'm not sure of their difference in switching library versions.

Reply:
I was not sure too... `CUDART_VERSION` seems get runtime version, while `__CUDACC_VER_MAJOR__ `  is defined with the major version number of nvcc when compiling. Thus, I agree `CUDART_VERSION` is better, and I will switch to `CUDART_VERSION`, also in consistent with other places in the code.